src/orders/components/OrderInvoiceEmailSendDialog/OrderInvoiceEmailSendDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import ConfirmButton, {
+  ConfirmButtonTransitionState
+} from "@saleor/components/ConfirmButton";
+import FormSpacer from "@saleor/components/FormSpacer";
+import { buttonMessages } from "@saleor/intl";
+import { InvoiceErrorFragment } from "@saleor/orders/types/InvoiceErrorFragment";
+import { InvoiceFragment } from "@saleor/orders/types/InvoiceFragment";
+import getInvoiceErrorMessage from "@saleor/utils/errors/invoice";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+export interface OrderInvoiceEmailSendDialogProps {
+  confirmButtonState: ConfirmButtonTransitionState;
+  errors: InvoiceErrorFragment[];
+  open: boolean;
+  invoice: InvoiceFragment;
+  onClose: () => void;
+  onSend: () => void;
+}
```

Comment:
Let's keep things tidy

```suggestion
export interface OrderInvoiceEmailSendDialogProps extends DialogProps {
  confirmButtonState: ConfirmButtonTransitionState;
  errors: InvoiceErrorFragment[];
  invoice: InvoiceFragment;
  onSend: () => void;
}
```

Reply:
fixed